### ISSUE TITLE
Latest Tag doesn't contain php-fpm

### ISSUE THREAD

**Issue by Knight1** [diginc/docker-pi-hole#127]
Hey,

the latest tag at hub.docker.com doesn't contain php-fpm.

```/bash_functions.sh: line 173: php-fpm: command not found```

Hope you can fix this ASAP 👍 
Tobias

**Comment by diginc** [comment]
The github magic auto-closed this...still waiting on docker cloud to build the image though

![image](https://cloud.githubusercontent.com/assets/868002/26074784/be27eaee-3978-11e7-9a89-ebe289d8e34c.png)

**Comment by diginc** [comment]
OK the build finally ran.  pulled the new latest tag and it looks happy now:

```
$ docker exec pihole_old ps -ef | grep php
   48 root       0:00 {php-fpm5} php-fpm: master process (/etc/php5/php-fpm.conf)
   49 nginx      0:00 {php-fpm5} php-fpm: pool www
   50 nginx      0:00 {php-fpm5} php-fpm: pool www
```

The github hook also ran a debian and arm build.  I tried to cancel them (which failed) to speed up the builds but looks like they built & pushed new versions anyway.  There shouldn't be any issue, I can some quick tests that they startup & web interface loads.

This happened because the readme for the new 3.0 version got released early when docker hub/store published the :dev builds for the new 3.0 version.  I re-ran the old build to get the old README back up there.  I suppose I should have just manually edited the readme instead of trusting the build was reproducible, in my head I thought It'd use cached builds since I have those enabled on docker cloud.

**Comment by Knight1** [comment]
```
time="2017-05-15T19:40:00Z" level=info msg="Found new diginc/pi-hole:latest image (sha256:3bded47f8856cd37fdbda0e208d90550338a1dc44c617f539d2f3c64c165d688)"
time="2017-05-15T19:40:20Z" level=info msg="Stopping /pihole (1b262e9f7ef8b045fd35d36abc34f5191ef403626d61ad7e28f85b9bbecc242c) with SIGTERM"
time="2017-05-15T19:40:24Z" level=info msg="Creating /pihole"
time="2017-05-15T19:40:28Z" level=info msg="Removing image sha256:be11a26bfd3df44cbbac22fab40aa987f5c7a7064056c4813b667ca5ab93e4b6"
```
```
dig google.com @10.0.1.5 +short
216.58.206.14
```
Thank you @diginc :)
